### PR TITLE
Kuz-323 action delete rooms

### PR DESCRIPTION
**Pull request by AnthonySendra**
New action to remove all rooms in collection or listed rooms in collection

### CODE REVIEW

File: lib/api/core/hotelClerk.js [hunk 1]
```diff
@@ -225,11 +226,166 @@ module.exports = function HotelClerk (kuzzle) {
...
     return deferred.promise;
   };
+
+  /**
+   * Remove rooms for a given collection
+   * If rooms attribute is not provided, all rooms for the collection are removed
+   * @param {RequestObject} requestObject
+   * @returns {Promise} resolve a response object. May have PartialError if rooms are provided and don't match collection
+   */
+  this.removeAllRooms = function (requestObject) {
```
Comment (scottinet) [inline lib/api/core/hotelClerk.js]: As this method may remove a selection of rooms, shouldn't it be called ``removeRooms`` instead?

**Comment by scottinet** [comment]
+1

**Comment by benoitvidis** [comment]
+1

File: lib/api/core/hotelClerk.js (hunk 1 above)
Comment (ballinette) [inline lib/api/core/hotelClerk.js]: +1

Comment (stafyniaksacha) [inline lib/api/core/hotelClerk.js]: +1

Comment (AnthonySendra) [inline lib/api/core/hotelClerk.js]: Done !

**Comment by ballinette** [comment]
+1

**Pull request merged**